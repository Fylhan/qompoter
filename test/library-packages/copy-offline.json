{
	"name": "install-ok/qompoter-versionstar-offline",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/acme": "v1.*-lib",
		"cp/fylhan3": "v1.0-lib"
	}
}
