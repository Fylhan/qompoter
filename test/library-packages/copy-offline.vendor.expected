vendor
vendor/acme
vendor/acme/qompoter.json
vendor/acme/qompoter.pri
vendor/fylhan3
vendor/fylhan3/qompoter.json
vendor/fylhan3/qompoter.pri
vendor/lib_linux_64
vendor/lib_linux_64/acme_fake.a
vendor/lib_linux_64/fylhan3_fake.a
vendor/qompote.pri
vendor/vendor.pri
